src/core/Akka.Persistence/Snapshot/SnapshotStore.cs

```diff
@@ -225,8 +238,11 @@ private Exception TryUnwrapException(Exception e)
         /// </summary>
         /// <param name="persistenceId">Id of the persistent actor.</param>
         /// <param name="criteria">Selection criteria for loading.</param>
-        /// <returns>TBD</returns>
-        protected abstract Task<SelectedSnapshot> LoadAsync(string persistenceId, SnapshotSelectionCriteria criteria);
+        /// <param name="cancellationToken"><see cref="CancellationToken"/> used to signal cancelled snapshot operation</param>
+        protected abstract Task<SelectedSnapshot> LoadAsync(
+            string persistenceId, 
+            SnapshotSelectionCriteria criteria,
+            CancellationToken cancellationToken);
```

Comment:
[Breaking change] `Akka.Persistence` codes that are being wrapped inside a `CircuitBreaker` will now take a `CancellationToken` as a signal that the operation has been timed out by the `CircuitBreaker`.